src/main/kotlin/uk/gov/justice/digital/hmpps/hmppstier/domain/Registrations.kt

```diff
@@ -0,0 +1,13 @@
+package uk.gov.justice.digital.hmpps.hmppstier.domain
+
+import uk.gov.justice.digital.hmpps.hmppstier.client.Registration
+import uk.gov.justice.digital.hmpps.hmppstier.domain.enums.ComplexityFactor
+import uk.gov.justice.digital.hmpps.hmppstier.domain.enums.Mappa
+import uk.gov.justice.digital.hmpps.hmppstier.domain.enums.Rosh
+
+data class Registrations(
+  val iomNominal: List<Registration>,
```

Comment:
This might as well be a single registration, and simplifying it further it might as well be a Boolean, it depends where we want the business logic to live

Reply:
I was thinking the same! Didn't get that far this time but it is the kind of thing which could live in the client